tests/testfiles/CommonMark_0.30/html_blocks/001.test

```diff
@@ -23,6 +23,7 @@ documentBegin
 inputBlockHtmlElement: ./_markdown_test/5a8e413f9b0ef97790d0c9da301a6149.verbatim
 interblockSeparator
 emphasis: world
+inlineHtmlTag: </pre>
```

Comment:
In inline HTML content, we only recognize tags rather than entire elements.
See <https://witiko.github.io/markdown/#html-tag-and-element-renderers>.